students/johnpharmd/lesson03/list_lab.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python3
+
+
+def get_fruits():
+    """
+    Create a list that contains “Apples”, “Pears”, “Oranges” and “Peaches”.
+    """
+    fruits = ['Apples', 'Pears', 'Oranges', 'Peaches']
```

Comment:
You could just declare `fruits` as a global variable and then pass it in to your series functions below

Reply:
Although I should note that if this module were part of a larger system, you would want to be very careful about defining variables in the global namespace, so the way you're doing this here is fine as well